packages/test-app/package.json

```diff
@@ -15,7 +15,7 @@
     "@scalprum/core": "*",
     "@scalprum/react-core": "*",
     "@types/history": "^4.7.9",
-    "@types/react": "^17.0.31",
+    "@types/react": "^18.0.0",
     "@types/react-dom": "^17.0.10",
```

Comment:
@Hyperkid123 since we touched react-dom, can we upgrade the associated type package to `^18.0.0"? Version `@types/react-dom@18.2.14` appears to be present in the NPM registry.